source/tools/aws-cdk-migration/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Are we duplicating the cdk migration tool here?

Why would we do this?

Reply:
Unfortunately yes, for the same reason as cdk-integ-tools, it needed to be modified to handle special case of rewriting the import statments for `IConstruct`